.gitignore

```diff
@@ -1,2 +1,3 @@
 /target
 /Cargo.lock
+/output
```

Comment:
what is writing to this directoty?

Reply:
if not of our code write into output, remove this line, the code in general should not have refences to thigns that you are doing only in local

Reply:
Done!